third_party/istio-1.4-latest

```diff
@@ -1 +1 @@
-istio-1.4.2
\ No newline at end of file
```

Comment:
<!--whitespace-bot-->
Add trailing newline:
```suggestion
istio-1.4.4

```

Reply:
This is a symlinc so no newline is alright.